src/instances/arr/index.js

```diff
@@ -117,6 +120,8 @@ const Arr = (() => {
   // :: Monoid m -> (a -> m) -> [a] -> m
   const foldMap = M => f => foldr(a => M.append(f(a)))(M.empty);
 
+  const fold = M => foldMap(M)(x => x);
```

Comment:
Can we add a TODO here to solve this problem at some future date (like, having a common definition of `fold`).